collections/src/main/scala/strawman/collection/IterableOnce.scala

```diff
@@ -55,24 +55,24 @@ final class IterableOnceExtensionMethods[A](private val it: IterableOnce[A]) ext
   def toList: immutable.List[A] = immutable.List.from(it)
 
   @deprecated("Use Set.from(it) instead of it.toSet", "2.13.0")
-  @`inline` final def toSet[B >: A]: immutable.Set[B] = immutable.Set.from(it)
+  @`inline` def toSet[B >: A]: immutable.Set[B] = immutable.Set.from(it)
```

Comment:
Why did you un-finalize these methods? Note that `@inline` only works on final methods.

Reply:
Because the outer class is already final.